dynamodb-enhanced/runtime/src/main/java/io/quarkus/amazon/dynamodb/enhanced/runtime/DynamodbEnhancedFeature.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.amazon.dynamodb.enhanced.runtime;
+
+import org.graalvm.nativeimage.hosted.Feature;
+
+import com.oracle.svm.core.annotate.AutomaticFeature;
+import org.graalvm.nativeimage.hosted.RuntimeReflection;
+import software.amazon.awssdk.enhanced.dynamodb.DefaultAttributeConverterProvider;
+import software.amazon.awssdk.enhanced.dynamodb.internal.mapper.BeanTableSchemaAttributeTags;
+
+@AutomaticFeature
+public class DynamodbEnhancedFeature implements Feature {
```

Comment:
This could be removed, see comment in EnhancedProcessor

Reply:
Yes, this should be removed and be in the processor.